frontend/src/pages/AccountSend.vue

```diff
@@ -1206,16 +1222,15 @@ function useSendForm() {
     );
   }
 
-  function checkConfusables() {
-    const recipientIdString = recipientId.value || '';
+  function checkConfusables(recipientIdString: string | undefined, index: number | undefined) {
     try {
       if (recipientIdString && recipientIdString.endsWith('eth')) {
         assertValidEnsName(recipientIdString);
       }
-      recipientIdWarning.value = undefined;
-    } catch (e) {
-      if (e instanceof Error) {
-        recipientIdWarning.value = e.message;
+      setWarning('', index); // clear warning
+    } catch (e: unknown) {
```

Comment:
I think this `unknown` can be removed. Explicit type declarations can cause issues down the line if they are un-needded